hyperspy/_signals/eels.py

```diff
@@ -751,6 +751,107 @@ def fourier_ratio_deconvolution(self, ll,
                 self.tmp_parameters.filename +
                 'after_fourier_ratio_deconvolution')
         return cl
+    
+    def fourier_ratio_deconvolution_llspectra(self, kernel, zl=None,
+                                        fwhm=None,
+                                        threshold=None):
+        """Performs Fourier-ratio deconvolution of two similar spectrum.
+        
+        Parameters
+        ----------
+        kernel: EELSSpectrum
+            The spectrum to be deconvolved from parent spectrum
+        zl: ZLP spectrum to avoid Gaussian aproximation of ZLP
```

Comment:
Also, should be a space between the keyword and the explanation. So "zl : ZLP..."